src/main/drivers/accgyro_mpu.c

```diff
@@ -241,254 +223,15 @@ void MPU_DATA_READY_EXTI_Handler(void)
         uint32_t callDelta = now - lastCalledAt;
 		debug[0] = callDelta;
 		lastCalledAt = now;
-/*		if (callDelta > 150) {
-			if (timewatchdog < 32) {
-				timewatch[timewatchdog] = 1;
-				timewatchdog++;
-			}
-		} else {
-			if (timewatchdog < 32) {
-				timewatch[timewatchdog] = 0;
-				timewatchdog++;
-			}
-		}
-		if (timewatchdog == 32) {
-			debug[2]=timewatch[0] << 0 | timewatch[1] << 1 | timewatch[2] << 2  | timewatch[3] << 3  | timewatch[4] << 4  | timewatch[5] << 5  | timewatch[6] << 6  | timewatch[7] << 7 | timewatch[8] << 8 | timewatch[9] << 9 | timewatch[10]<< 10 | timewatch[11]<< 11 | timewatch[12]<< 12 | timewatch[13]<< 13 | timewatch[14]<< 14 | timewatch[15]<< 15;
-			debug[3]=timewatch[16]<< 0 | timewatch[17]<< 1 | timewatch[18]<< 2  | timewatch[19]<< 3  | timewatch[20]<< 4  | timewatch[21]<< 5  | timewatch[22]<< 6  | timewatch[23]<< 7 | timewatch[24]<< 8 | timewatch[25]<< 9 | timewatch[26]<< 10 | timewatch[27]<< 11 | timewatch[28]<< 12 | timewatch[29]<< 13 | timewatch[30]<< 14 | timewatch[31]<< 15;
-			debug[1]++;
-			timewatchdog=0;
-		}
-*/
 #endif
```

Comment:
Yes, I probably have other code that's just commented out that needs to go. It's a bad habit.